src/services/events/events-router.ts

```diff
@@ -83,4 +100,22 @@ eventsRouter.delete("/:EVENTID", async (req, res, next) => {
     }
 });
 
+eventsRouter.post("/check-in", async (req, res, next) => {
+    try {
+        const { eventId, userId } = req.body;
+        const result = await checkInUser(eventId, userId);
+        if (result.success) {
+            return res
+                .status(StatusCodes.OK)
+                .json({ message: "Check-in successful" });
+        } else {
+            return res
+                .status(StatusCodes.NOT_FOUND)
+                .json({ error: result.message });
+        }
+    } catch (error) {
+        next(error);
+    }
+});
+
```

Comment:
no need for this

Reply:
okay i left this for testing, like the qr function; can get rid of later